Tribler/Test/Core/TorrentChecker/test_torrentchecker_session.py

```diff
@@ -275,6 +275,17 @@ def on_error(failure):
 
         return test_deferred
 
+    import mock
+
+    def dummy_is_pending_task_active(*args, **kwargs):
+        return True
+
+    @mock.patch("Tribler.Core.TorrentChecker.session.UdpTrackerSession.is_pending_task_active",
```

Comment:
I will do that. Actually this isn't working and I just wanted to save it at the end of the day.
(I didn't know that it was updated here automatically after I pushed it to my own repository.)